modules/core/04-channel/types/events.go

```diff
@@ -39,7 +39,11 @@ const (
 	AttributeKeyChannelOrdering  = "packet_channel_ordering"
 	AttributeKeyConnection       = "packet_connection"
 
-	AttributeKeyUpgradeSequence = "upgrade_sequence"
+	// upgrade specific keys
+	AttributeKeyUpgradeSequence       = "upgrade_sequence"
+	AttributeKeyUpgradeVersion        = "upgrade_version"
+	AttributeKeyUpgradeConnectionHops = "upgrade_connection_hops"
+	AttributeKeyUpgradeOrder          = "upgrade_order"
```

Comment:
nit but aligns more so with `AttributeKeyChannelOrdering` 

```suggestion
	AttributeKeyUpgradeOrder          = "upgrade_ordering"
```